src/components/filters/FilterSelect.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {Select} from "antd";
+import "antd/es/select/style/css";
+
+const { Option } = Select;
+
+const FilterSelect = ({
+  filterType,
+  filterTypeName,
+  options,
+  mode,
+  placeholder,
+  onChange,
+  filters
+}) => {
+  const convertFiltersToArr = (filters) => {
+    return typeof(filters) == 'string' ? filters.split(",") : filters
+  }
```

Comment:
Notes:
 - Doesn't need to be inside this function, could be defined at the bottom of the file as `function toArray(...) ...`. Keeping small helpers as functions at the bottom of the file allows the code here to be focused in the core business logic rather than implementation details.
 - Used only once, can be inlined.
 - Most of the time, `typeof` is a code smell